adafruit_minimqtt/adafruit_minimqtt.py

```diff
@@ -398,18 +431,15 @@ def ping(self):
         there is an active network connection.
         """
         self.is_connected()
+        if self._pending_ping_response:
+            self._pending_ping_response = False
+            raise MMQTTException("Ping response was pending from previous MQTT_PINGREQ")
```

Comment:
Don't raise an exception here, raise one within the method which calls `ping()`

Reply:
why?